src/app/services/fractalis.service.ts

```diff
@@ -64,23 +62,20 @@ export class FractalisService {
               private cohortService: CohortService,
               private resourceService: ResourceService) {
     this.chartDivSize = 35;
-
-    if (!this.appConfig.getConfig('enable-fractalis-analysis')) {
-      MessageHelper.alert('warn', 'Fractalis analysis is disabled.');
-    } else if (!fjs.fractalis) {
+    const enabled = this.appConfig.getConfig('enable-fractalis-analysis');
+    if (enabled && !fjs.fractalis) {
       MessageHelper.alert('error', 'Failed to import Fractalis.');
     } else {
-      this.setupFractalis();
-      this.selectedVariablesUpdated.asObservable().subscribe(variables => {
-        this.prepareCache(variables);
-      });
       this.timer = setInterval(() =>
         this.updateVariablesStatus(), FractalisService.TIMEOUT_VARIABLE_STATUS_UPDATE);
```

Comment:
This already starts polling the variable statuses and causes an error, because the `F` object is not yet defined.